charts/opentelemetry-collector/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## OpenTelemetry Collector
 
+### v0.98.5 / 2024-11-28
+- [Feat] Adding new configs to the Target Allocator.
+
```

Comment:
@nicolastakashi do you mind adding a changelog entry here for v0.98.4, please? I forgot to add it on #121. :(

It should be something like:

```
### v0.98.4 / 2024-11-28
- [Fix] Make the metrics telemetry service listen on `0.0.0.0` instead of using shell var expansion to resolve the pod IP.
```